src/proguard/proguard.targets

```diff
@@ -0,0 +1,37 @@
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="_BuildProGuard">
+    <Exec
+        Command="$(AntDirectory)\bin\ant -verbose -f buildscripts\build.xml proguard"
+        WorkingDirectory="$(ProGuardSourceFullPath)"
+    />
+    <MakeDir Directories="$(OutputPath)" />
+    <Copy
+        SourceFiles="..\..\external\proguard\lib\proguard.jar"
+        DestinationFiles="$(OutputPath)\lib\proguard.jar"
+    />
+    <Copy
+        SourceFiles="..\..\external\proguard\bin\proguard.bat"
+        DestinationFiles="$(OutputPath)\bin\proguard.bat"
+    />
+    <Copy
+        SourceFiles="..\..\external\proguard\bin\proguard.sh"
+        DestinationFiles="$(OutputPath)\bin\proguard.sh"
+    />
+    <Copy
+        SourceFiles="..\..\external\proguard\docs\license.html"
+        DestinationFiles="$(OutputPath)\license.html"
+    />
+  </Target>
+  <Target Name="_CleanProGuard"
+      AfterTargets="Clean">
```

Comment:
Next time you are going to make similar changes, it would be nicer to use $(CleanDependsOn) than using AfterTargets that you cannot control the target execution order.

It's not a big deal here so I'm not going to block it because of this.